docs/src/Module.doc.js

```diff
@@ -159,4 +176,71 @@ function ModuleExample4() {
   />
 );
 
+card(
+  <Example
+    name="Example with external control"
+    defaultCode={`
+function ModuleExample5() {
+  const [extExpandedId, setExtExpandedId] = React.useState(null);
+  const mapIds = {
+      'first-0': 0,
+      'first-1': 1,
+      'second-0': 0,
+      'second-1': 1,
+  }
+  return (
+    <Box maxWidth={800} padding={2} column={12}>
+      <Flex direction='column' gap={3}>
+        <Box padding={1} borderStyle='sm'>
+          <Text>Step 1</Text>
+          <Module.Expandable
+            id="ModuleExampleStep1"
+            accessibilityExpandLabel="Expand the module"
+            accessibilityCollapseLabel="Collapse the module"
+            expandedIndex={extExpandedId && extExpandedId.startsWith('first') && mapIds[extExpandedId]}
+            onExpandedChange={(index) => setExtExpandedId('first-'+index)}
```

Comment:
As written, this will update `extExpandedId` to `'first-null'` when all module items are collapsed. While this functions correctly, it's not technically correct, and shouldn't be used as a pattern. (Also, we should avoid using the `+` for string concatenation.)

Consider something like
```js
(index) => setExtExpandedId(Number.isFinite(index) ? `first-${index}` : index)
```